engine-comparison/runner.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Copyright 2017 Google Inc. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+
+. metadata.cfg
+. parameters.cfg
+. fengine.cfg
+
+BINARY=${BENCHMARK}-${FUZZING_ENGINE}
+mkdir -p corpus
+chmod 750 $BINARY
+
+if [[ FUZZING_ENGINE == "afl" ]]; then
+  chmod 750 afl-fuzz
+  BINARY="afl-fuzz $BINARY"
+fi
+
+./$BINARY $BINARY_RUNTIME_OPTIONS # -workers=$JOBS -jobs=$JOBS -artifact_prefix=corpus
+
+mkdir -p results
+while [[ ! -e results/complete.txt ]] : do
+  sleep 12
+  # TODO: create a copy fuzz-0.log here; the copy can be read to reduce errors
```

Comment:
With libfuzzer, its sufficient to read one log if `-artifact_prefix=` is assigned, because each process will access any new coverage found by another process (a fine point, I realized that this is crucial for every `test-libfuzzer.sh` in this repo). However, we don't assign `-artifact_prefix` by default, and I'm not too familiar with AFL logging yet either.

To the next point, I was thinking that I wouldn't want to read the log file while it was being written to at the same time, because this could be some sort of pseudo race condition.

In any case, this code is likely a moot point; I discussed this with Kostya, and it seems that all relevant data should be extracted directly from measurements of the directory containing corpus artifacts, instead of parsing a log file (I don't think AFL has such robust logging)